project_euler/problem_56/sol1.py

```diff
@@ -1,15 +1,27 @@
-def maximum_digital_sum(a,b):
-    '''
-    Considering natural numbers of the form, a**b, where a, b < 100,
-    what is the maximum digital sum?
-    :param a:
-    :param b:
-    :return:
-    '''
+import typing
+
+
+def maximum_digital_sum(a,b) -> int:
```

Comment:
Can __a__ and __b__ only be __ints__ or could they be __floats__ as well?  Same question on the __return__ value.

Reply:
**a** and **b** can be only **int**.

Reply:
(a: int,b: int) -> int: